### PR TITLE
Add optional unsanitized html support

### PR DESCRIPTION
**Pull request by Tankenstein**
Done intentionally using a nasty `dangerouslyTranslateInnerHTML` property

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,11 +25,13 @@ const translations = {
     'main.heading': 'retranslate #{{ versionNumber }}',
     'main.subtitle': 'Real simple translations for react.',
     'current.language': 'Your current language is {{ currentLanguage }}',
+    'bold.thing': 'This <b>thing</b> is bold',
```
Comment (skywritergr) [inline README.md]: I wouldn't name it thing. Maybe text?

File: package.json [hunk 2]
```diff
@@ -89,9 +89,13 @@
...
   "eslintConfig": {
     "extends": "airbnb",
+    "env": {
+      "jest": true
+    },
     "rules": {
       "react/jsx-filename-extension": 0,
-      "import/prefer-default-export": 0
+      "import/prefer-default-export": 0,
+      "import/no-extraneous-dependencies": 0
```
Comment (skywritergr) [inline package.json]: trailing comma?

File: src/message/Message.js [hunk 3]
```diff
@@ -1,13 +1,27 @@
 import React, { PropTypes } from 'react';
 
-const Message = ({ children, params, className }, { translations: { translate } }) => (
-  <span className={className}>{translate(children, params)}</span>
-);
+const Message = (
```
Comment (skywritergr) [inline src/message/Message.js]: I think this part is complicated for future maintainers (possibly?). Destructuring and arrow functions all at once can be confusing for people less familiar. But it's not wrong by any means.

File: README.md (hunk 1 above)
Comment (Tankenstein) [inline README.md]: Sure

File: package.json (hunk 2 above)
Comment (Tankenstein) [inline package.json]: Trailing commas are not valid in JSON, this is JSON not javascript :)

File: src/message/Message.js (hunk 3 above)
Comment (Tankenstein) [inline src/message/Message.js]: Eh, just a matter of time until everyone knows es2015 :)

**Pull request merged**